{{cookiecutter.repostory_name}}/envs/prod/docker-compose.yml

```diff
@@ -107,22 +107,17 @@ services:
   {% endif -%}
 
   nginx:
-    image: 'nginx:stable-alpine'
+    image: 'ghcr.io/reef-technologies/nginx-rt:master'
     restart: unless-stopped
     healthcheck:
-      test: wget -q --spider 0.0.0.0:80/admin/login/ || exit 1
+      test: wget -q --spider 0.0.0.0:80 || exit 1
```

Comment:
it was correct - admin is always there, `/` often times isn't

Reply:
No, because with this new setup, we have a blank page on 0.0.0.0 (see default http)